src/Fields/field.jl

```diff
@@ -99,7 +99,7 @@ function Field(loc::Tuple, grid::AbstractGrid, data, bcs, indices, op=nothing, s
     @apply_regionally indices = validate_indices(indices, loc, grid)
     @apply_regionally validate_field_data(loc, data, grid, indices)
     @apply_regionally validate_boundary_conditions(loc, grid, bcs)
-    buffers = FieldBoundaryBuffers(grid, data, bcs)
+    buffers = nothing
```

Comment:
maybe this can be a function that we extend in the distributed module if the goal eventually is to remove the special field constructor for distributed

Reply:
we can do that, but why do we need this if we extend Field?

Reply:
oh sorry I read what you wrote too quickly. Yes it would be easy to define `communication_buffer` which returns `nothing` or `CommunicationBuffer`. I think it'd be better to implement that when we do in fact eliminate the special field constructor, rather than anticipating it here.